src/lightning/app/launcher/launcher.py

```diff
@@ -0,0 +1,480 @@
+import inspect
+import logging
+import os
+import signal
+import sys
+import time
+import traceback
+from functools import partial
+from multiprocessing import Process
+from typing import Callable, Dict, List, Optional, Tuple, TypedDict
+
+from lightning_utilities.core.imports import module_available
+
+ENABLE_MULTIPLE_WORKS_IN_DEFAULT_CONTAINER = bool(int(os.getenv("ENABLE_MULTIPLE_WORKS_IN_DEFAULT_CONTAINER", "0")))
+
+
+if module_available("lightning_app"):
```

Comment:
this is not needed, the switch is on package level when we build it, you can see other commands...